### DISCUSSION TITLE
학습시 label을 어떤 value로 하는 것이 좋을 지

### DISCUSSION THREAD

**Discussion by leecs0503** [leecs0503/NLP_FinalProject#15] [Ideas]
1. 10개의 라벨 중 가장 많이 나온 라벨로 선택
2. valid한 라벨 중 랜덤으로 선택
3. 10개의 라벨 중 랜덤으로 선택

각 방식의 예상되는 장단점을 말씀주시고, 추가적인 좋은 아이디어가 있으면 공유주세요.

**Comment by kig9981** [comment]
일단 논문 기반이니까 valid한 라벨 중 고르는건 맞는거같은데, all_answer에서 특정 답이 일정 비율 이상(예를 들어 과반수)일 경우 나머지 대답은 noise인걸로 생각하고 그 답을 answer_label로 고르고, 아니면 랜덤으로 하는건 어떤가요?
